bin/crx.js

```diff
@@ -23,6 +23,11 @@ program.version(pkg.version);
 program
   .command("keygen [directory]")
   .option("--force", "overwrite the private key if it exists")
+  .option(
+    "-f, --format-version [number]",
+    "CRX format version, can be either 2 or 3",
```

Comment:
Mention that it defaults to 3. Also maybe we should use `-v`/`--version` here? `-f` is pretty easy to mix up with `--force`.

Reply:
Good catch, thanks. I changed it to `-c, --crx-format`, because `-v`/`--version` is most often used as a way to get application's version, not set one.